zhaquirks/tuya/ts0041.py

```diff
@@ -103,3 +103,80 @@ class TuyaSmartRemote0041TI(CustomDevice):
         (LONG_PRESS, BUTTON_1): {ENDPOINT_ID: 1, COMMAND: LONG_PRESS},
         (DOUBLE_PRESS, BUTTON_1): {ENDPOINT_ID: 1, COMMAND: DOUBLE_PRESS},
     }
+
+
+class TuyaSmartRemote0041TOPlusA(CustomDevice):
+    """Tuya 1-button remote device with time on out cluster."""
+
+    signature = {
+        # SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=0, device_version=1, input_clusters=[0, 1, 6, 57344], output_clusters=[10, 25]))
+        # SizePrefixedSimpleDescriptor(endpoint=2, profile=260, device_type=0, device_version=1, input_clusters=[1, 6], output_clusters=[])
+        # SizePrefixedSimpleDescriptor(endpoint=3, profile=260, device_type=0, device_version=1, input_clusters=[1, 6], output_clusters=[])
+        # SizePrefixedSimpleDescriptor(endpoint=4, profile=260, device_type=0, device_version=1, input_clusters=[1, 6], output_clusters=[])
```

Comment:
One last thing, I think these comments can be removed (there's basically no added information they provide that the quirk doesn't already contain)

Reply:
For the sake of consistency I'd keep them because other quirks in this and other "tuya" files have similar comments.
Or remove everywhere )

Reply:
I see, yeah. 👍